### ISSUE TITLE
Not charging when max current set to +6A

### ISSUE THREAD

**Issue by Steven1994** [leeyuentuen/alfen_wallbox#138]
I am setting up my charging station to switch so I can optimally charge the car. I have an automation where I change the max current of my power connector based on my injection and night tariff. The car was already plugged in, and just now at 21:00, I set the max current to 6A and it worked (see first image). However, the car only started charging at 21:35 (second image).

I have also attached the logs, the log from 21:00 when the current was set to 6A, and the log from 21:37 when it actually started charging. On the screen, between 21:00 and 21:37, the charging station displayed (waiting for vehicle to charge), and at 21:37 it suddenly started charging.

What can be the problem that my car doesn't start charging when i give power to the wallbox?

![Display wallbox](https://github.com/user-attachments/assets/97433e76-2aff-4b91-b38e-5844d0694029)
![charging power](https://github.com/user-attachments/assets/cea5c5dc-b511-43f9-a292-efde29a6c4fd)
![current setting](https://github.com/user-attachments/assets/9ca0376b-97fb-4188-b2d6-7b78c16bc806)
![log 21h00](https://github.com/user-attachments/assets/f51cf09b-6c9b-4f84-89c3-980eba72d78e)
![log 21h37](https://github.com/user-attachments/assets/e8605b47-fb92-4cf5-a808-e718f11e5a3f)

**Comment by leeyuentuen** [comment]
what are the value of  charging mode?
maybe it was load balancing?

**Comment by Steven1994** [comment]
What do you mean with value of charging mode? Normally load balancing is not on by default in the wallbox, if it is on, then i want to turn it off as i want to control via HA, how do i check if it is on and turn it off?

**Comment by leeyuentuen** [comment]
there are few settings. for load balancing if you have the license on it, then it will be set by default. 

if you can turn off, i'l not sure but you can set the value higher wallbox_load_balancing_max_meter_current

they also check, if you are on 1 phase to not overload that phase,
maybe post your setting here to have more clear (but hide your personal information like wallbox id and tag id)

**Comment by Steven1994** [comment]
No, don't have the license for it, and will post settings here tonight when i am home, and what settings do you mean, the text file from the custom components?

**Comment by leeyuentuen** [comment]
something like on readme like this

![image](https://github.com/user-attachments/assets/558f8508-dd28-4f7a-8246-2d0b74f97ef4)

**Comment by Steven1994** [comment]
I noticed today that if I switch between for example 6A and 8A it reacts immediately. And if I switch the first time from after connecting the car from 0A to 6A it also works, only if I switch to 0A and ghen back to 6A, then it doesnt work. 
![Screenshot_2024-08-01-00-04-21-21](https://github.com/user-attachments/assets/ba574fae-7f64-4629-9525-4f9338e2e20f)

**Comment by Steven1994** [comment]
@leeyuentuen, can you see something that can be the issue? This is what is on the screen when it doesn't want to start charging
![IMG20240730210151](https://github.com/user-attachments/assets/f9ca9f4c-06ea-4b3f-b0ae-7ff759b4f4ca)

**Comment by leeyuentuen** [comment]
seems the setting are nothing special. i think for finding the real reason, you need to use ace installer and look thr whole log to see what happened.

**Comment by fwestenberg** [comment]
To me it seems your car is still waiting for more current. As the display also tells, the charger is ready. This is not related to the integration imo.
